fili-core/src/test/groovy/com/yahoo/bard/webservice/web/endpoints/BaseDataServletComponentSpec.groovy

```diff
@@ -71,6 +72,10 @@ abstract class BaseDataServletComponentSpec extends Specification {
         populatePhysicalTableAvailability()
     }
 
+    MultivaluedHashMap<String, String> getAdditionalHeaders() {
```

Comment:
It's not clear what these headers are for without tracing how this is used. Perhaps we should expand the method name to something like `getAdditionalApiRequestHeaders`? A JavaDoc would also help, but not critical if the method name is clearer.